django-backend/fecfiler/openfec/test_efo_mock_data/committee_accounts.json

```diff
@@ -2,46 +2,244 @@
     {
         "committee_id": "C00100214",
         "name": "TEST_COMMITTEE_NAME3",
-        "committee_type_full": "TEST_COMMITTEE_TYPE3"
+        "committee_type_full": "TEST_COMMITTEE_TYPE3",
+        "street_1": "TEST_SA3",
+        "city": "TEST_CITY3",
+        "state": "AL",
+        "zip": "12345",
+        "email": "gregg.morelenad@tcg.com",
```

Comment:
It's probably not an issue, but it would be nice to avoid putting people's emails in the repo

Reply:
Great suggestion.  This is used to pre-populate the email during filing and can easily be replaced.  Changed these in the test .json.